compiler/testData/diagnostics/tests/DeprecatedGetSetPropertyDelegateConvention.fir.kt

```diff
@@ -33,8 +33,8 @@ operator fun CustomDelegate3.setValue(thisRef: Any?, prop: KProperty<*>, value:
 
 class Example {
 
-    var a by <!INAPPLICABLE_CANDIDATE!>CustomDelegate<!>()
-    val aval by <!INAPPLICABLE_CANDIDATE!>CustomDelegate<!>()
+    var a by <!DELEGATE_SPECIAL_FUNCTION_NONE_APPLICABLE!>CustomDelegate()<!>
+    val aval by <!DELEGATE_SPECIAL_FUNCTION_NONE_APPLICABLE!>CustomDelegate()<!>
```

Comment:
These are `...MISSING` instead: https://github.com/JetBrains/kotlin/blob/master/compiler/testData/diagnostics/tests/DeprecatedGetSetPropertyDelegateConvention.kt#L37

Also, for the first one, FE 1.0 reports twice, and I wonder why it did as well as why we don't.

Reply:
Thanks! This is same as https://github.com/JetBrains/kotlin/pull/4287#discussion_r609055104

Reply:
This is now resolved with commit#1.